### PR TITLE
Add input to ref

### PR DESCRIPTION
**Pull request by li4e**

### CODE REVIEW

File: lib/Input.js [hunk 1]
```diff
@@ -192,13 +192,18 @@ export default class extends Component {
           onChangeText={this._handleChangeText}
           onContentSizeChange={this._handleContentSizeChange}
           value={value}
+          ref={input => (this.input = input)}
```
Comment (egorkhmelev) [inline lib/Input.js]: Lets use handler function and private variable should be underscored

Comment (li4e) [reply lib/Input.js]: Done

File: lib/Input.js [hunk 2]
```diff
@@ -192,13 +192,20 @@ export default class extends Component {
...
           value={value}
+          ref={this._handleTextInputRef}
         />
         <Underline {...underlineProps} />
         {error ? <ErrorHelper {...errorProps} /> : null}
       </View>
     )
   }
 
+  _handleTextInputRef = input => (this._input = input)
```
Comment (egorkhmelev) [inline lib/Input.js]: Lets add `{ }`

File: lib/Input.js [hunk 3]
```diff
@@ -192,13 +192,20 @@ export default class extends Component {
...
+
+  focus = () => {
+    this._input.focus()
```
Comment (egorkhmelev) [inline lib/Input.js]: Its good idea to check for `this._input` availability before calling focus

**Review by egorkhmelev — CHANGES_REQUESTED** [review]

**Review by egorkhmelev — APPROVED** [review]

**Pull request merged**